web/src/utils/api.ts

```diff
@@ -27,6 +31,71 @@ const getBaseUrl = () => {
   return `http://localhost:${process.env.PORT ?? 3000}`; // dev SSR should use localhost
 };
 
+let buildVersion: string | null = null;
```

Comment:
I'd use `id` instead of `version`. adding a one-line comment to this variable makes sense to highlight how we use it and why this is necessary